### PR TITLE
Add new Landing Page

### PR DESCRIPTION
**Pull request by MrAizu**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
     <title>Document</title>
 </head>
 <body>
-    Welcome To Potionfy! You are Awesome 
+    Welcome To Potionfy! Enjoy your Stay
```
Comment (aizuDeann) [inline index.html]: Can you remove "Welcome To Potionfy! Enjoy your Stay" to "Welcome To Potionfy!" string red

Comment (MrAizu) [reply index.html]: Sure . I'll do it!

**Review by aizuDeann — CHANGES_REQUESTED** [review]
Please change the code as per comment review

**Review by aizuDeann — APPROVED** [review]
Well Done MrAizu

**Pull request merged**